diff --git a/qtbase/src/corelib/global/qlogging.cpp b/qtbase/src/corelib/global/qlogging.cpp
index 0000000..1111111 100644
--- a/qtbase/src/corelib/global/qlogging.cpp
+++ b/qtbase/src/corelib/global/qlogging.cpp
@@ -1290,6 +1290,7 @@
 
+#ifndef __ANDROID__
     int n = backtrace(buffer.data(), buffer.size());
     for (int i = 0; i < n; ++i) {
         QScopedPointer<char*, QScopedPointerPodDeleter> strings(backtrace_symbols(buffer.data() + i, 1));
@@ -1305,6 +1306,7 @@
         }
     }
+#endif
